src/Microsoft.ML.AutoML/TrainerExtensions/BinaryTrainerExtensions.cs

```diff
@@ -122,7 +122,7 @@ public IEnumerable<SweepableParam> GetHyperparamSweepRanges()
         public PipelineNode CreatePipelineNode(IEnumerable<SweepableParam> sweepParams, ColumnInformation columnInfo)
```

Comment:
I will just put the two ranking trainers into a separate `RankingTrainerExtensions` and wrap them into separate `ITrainerExtension`. Because ranking trainers come from a different catalog "`Ranking`", so it's better to differentiate them from BinaryTrainers to make it easier for CodeGen to generate the right pipeline